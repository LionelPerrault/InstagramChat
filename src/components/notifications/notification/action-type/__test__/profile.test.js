import React from 'react'
import { create } from 'react-test-renderer'
import NotificationTypeProfile from '../profile'

describe('NotificationTypeProfile Component', () => {
  it('should match snapshot', () => {
    const tree = create(
      <NotificationTypeProfile user_username="ghalib" />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
