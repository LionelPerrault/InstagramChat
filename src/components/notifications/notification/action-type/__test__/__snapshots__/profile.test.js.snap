// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationTypeProfile Component should match snapshot 1`] = `
<Link
  className="pri_btn"
  to="/profile/ghalib"
>
  View ghalib
</Link>
`;
